src/RefractDataStructure.cc

```diff
@@ -703,13 +738,7 @@ namespace drafter {
 
         if (!value.node->valueDefinition.values.empty() && (valuesCount != data.values.size())) {
```

Comment:
Yes it is. They are checking different conditions.
part `valuesCount != data.values.size()` saying, do this only in case  there are some values coming from TypeSections -> move first value into examples

It is posible to remove first part of condition